es/2/empty.md

```diff
@@ -1,6 +1,44 @@
 <!-- Add translation for the following page: https://vyper.fun/#/2/empty
 Do NOT change the code below. The below code runs the code editor -->
 
+# Capítulo 7: Función Empty
+
+En el capítulo anterior, se definió la lógica de una batalla pokemon y se retornó los detalles del pokemon salvaje si el entrenador pokemon gana.
+
+¿Pero qué pasa si el entrenador pokemon pierde? En tal caso necesitamos retornar algunos valores vacíos.
+
+## empty(typename) → Any
+
+`empty` es una función incorporada de Vyper que retorna un valor por defecto de tipo(`typename`) pasado como parámetro.
```

Comment:
Converting this line to English gives me the following translation:

> `empty` is a built -in function which returns a Vyper default type ( ` typename` ) as parameter.

Whereas the expected translation should give me:

> `empty` is a built-in Vyper function which returns a default value of the type(`typename`) passed as a parameter.

Your translation misses the "passed" word, which conveys that `empty` function returns the default value of the type that is passed as a parameter (if you pass a `String[64]` as a parameter, `empty` will give you the default value of `String[64]`).

The current translation fails to convey this. Please change it so that it conveys the correct purpose of the function  :smile:

Reply:
Perhaps this is a better approach:

> `empty` es una función integrada en Vyper que devuelve un valor por defecto del mismo tipo(`typename`) que haya sido pasado como parámetro.

Reply:
LGTM